.github/workflows/jekyll.yml

```diff
@@ -1,20 +1,76 @@
 name: Jekyll site CI
-
 on:
-  push:
-    branches: [ changes, release, translate ]
   pull_request:
-    branches: [ changes, release, translate ]
-
+    branches: [ next-release, release ]
+  push:
+    branches: [ next-release ]
 jobs:
-  build:
+  Jekyll_site_CI:
     permissions:
-      contents: read
+      contents: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Build the site in the jekyll/builder container
-      run: |
-        docker run \
-        -v ${{ github.workspace }}:/srv/jekyll -v ${{ github.workspace }}/_site:/srv/jekyll/_site \
-        jekyll/builder:latest /bin/bash -c "chmod a+w /srv/jekyll/Gemfile.lock && chmod 777 /srv/jekyll && jekyll build --future"
+      - uses: actions/checkout@v2
+
+# Retrieve po4a installation cache or create it if not found:
+      - name: Cache po4a dependencies
+        uses: actions/cache@v1.0.3
+        id: cache-po4a
+        with:
+          path: "~/po4a"
+          key: ${{ runner.os }}-po4a
+      - name: Install or retrieve po4a from cache
+        env:
+          CACHE_HIT: ${{steps.cache-po4a.outputs.cache-hit}}
+        run: |
+          if [[ "$CACHE_HIT" == 'true' ]]; then
+            sudo cp --force --recursive ~/po4a/* /
+          else
+            sudo apt install -yq gettext libsgmls-perl libyaml-tiny-perl opensp
+            wget -O po4a.deb https://github.com/jamulussoftware/assets/raw/main/po4a/po4a_0.64-alpha.deb
+            sudo dpkg -i po4a.deb
+            mkdir -p ~/po4a
+            for dep in po4a libcroco3 libosp5 sgml-base gettext libsgmls-perl libyaml-tiny-perl opensp; do
+                dpkg -L $dep | while IFS= read -r f; do if test -f $f; then echo $f; fi; done | xargs cp --parents --target-directory ~/po4a/
```

Comment:
Could you please explain what and why you’re doing this ? Do you check if all dependencies were installed correctly? Why is this needed?

Reply:
If `"$CACHE_HIT" == 'false'`, this creates a list of all the dependencies installed by the previous commands, required for a po4a installation, and copies them all to the cache folder (`~/po4a/`). This cache folder is what is stored by and retrieved later from git if `"$CACHE_HIT" == 'true'`.

Reply:
Ok. Leaving this open for other reviewers. Maybe you should add a comment in the code here.